src/lava/magma/core/run_configs.py

```diff
@@ -70,19 +70,21 @@ def select(self,
 class Loihi1SimCfg(RunConfig):
     """Run configuration selects appropriate ProcessModel -- either
     `SubProcessModel` for a hierarchical Process or else a `PyProcessModel`
-    for a standard Process.
+    for a standard Process. Implements and overrides `select()` method of the
+    parent class `RunConfig` to achieve this.
+
+    First, exceptions are checked in `exception_proc_model_map` dictionary
+    as key-value pairs `{Process: ProcessModel}`. These explicit
+    specifications are given precedence over the subsequent logic for
+    `ProcessModel` selection
 
     If any `SubProcessModel`s are available, then they will be returned first.
 
     In case of `PyProcessModel`s, an appropriate `PyProcessModel` is selected
     based on the tags associated with it. Tags are set using the `@tag`
     decorator. If no tags are set for `ProcessModel`s and no tag is selected,
     then the first matching `ProcessModel` will be returned.
-
-    If any exceptions are needed, they can be specified in
-    `exception_proc_model_map` dictionary as `{Process: ProcessModel}`
-    key-value pairs. These explicit specifications are given
-    precedence over the subsequent logic for `ProcessModel` selection"""
+    """
```

Comment:
Also document constructor of this class.

Reply:
Done.